lib/currency.rb

```diff
@@ -0,0 +1,29 @@
+module Currency
```

Comment:
Inacceptable. Thanks that you did not name it `Object` :)
Top-level should not be polluted, specifically with such a common named modules.

Should be:
```
module BusinessTime
  module Currency
```